product_variants_no_automatic_creation/views/product_view.xml

```diff
@@ -17,18 +17,54 @@
             <field name="model">product.template</field>
             <field name="inherit_id" ref="product.product_template_only_form_view" />
             <field name="arch" type="xml">
-                <button string="Variant Prices" position="attributes">
+            <xpath expr="//button[@name='103']" position="attributes">
```

Comment:
But this is because it's replaced in execution time by its current XML-ID. Here you can see the correct expression: https://github.com/odoo/odoo/blob/9.0/addons/product/product_view.xml#L77